relative_wallclock/relative_wallclock.py

```diff
@@ -0,0 +1,48 @@
+"""
+Adds an info-wallclock-relative command which prints the approximate wallclock time relative to the
+start of recording.
+    Usage: info-wallclock-relative
+"""
+
+import datetime
+import re
+
+import gdb
+
+from undo.debugger_extensions import (
+    debugger_utils,
+    udb,
+)
+
+
+class WallclockRelative(gdb.Command):
```

Comment:
You need a docstring here to get a message when you use `help`. Maybe move the file-level docstring here. Then test it with `help info-wallclock-relative` to make sure it's displayed OK.